docs/source/tracking.rst

```diff
@@ -180,6 +178,77 @@ and the `LocalArtifactRepository <https://github.com/mlflow/mlflow/blob/master/m
 `S3ArtifactRepository <https://github.com/mlflow/mlflow/blob/master/mlflow/store/artifact/s3_artifact_repo.py#L14>`_ are
 concrete implementations of the abstract class `ArtifactRepository <https://github.com/mlflow/mlflow/blob/master/mlflow/store/artifact/artifact_repo.py#L13>`_.
 
+Scenario 5: MLflow Tracking Server enabled as proxy for artifact storage access
+-------------------------------------------------------------------------------
+
+MLflow's Tracking Server supports utilizing the host as a proxy server for operations involving artifacts.
+Once configured with the appropriate access requirements, an administrator can start the tracking server to enable assumed-role operations involving the saving, loading, or listing of
+model artifacts, images, documents, and files. This eliminates the need to allow end users to have direct path access to a remote file store for artifact handling and eliminates the
+need for an end-user to provide access credentials to interact with an underlying file store.
+
+.. figure:: _static/images/scenario_5.png
```

Comment:
The "localhost" box in the `scenario_5` and `scenario_6` images show lines connecting from "RestStore" to "Remote Host. "RestStore" refers to the REST-based implementation of the backend metadata storage API, not an artifact storage API implementation. We should instead add a second small box referring to `HttpArtifactRepository` (https://github.com/mlflow/mlflow/blob/3c434246f73b41835194f3e6aa1f9be8488bcd56/mlflow/store/artifact/http_artifact_repo.py#L11).

Reply:
updated!